examples/nixnet/CAN_SIGNAL_SINGLE_POINT_OUTPUT.py

```diff
@@ -0,0 +1,87 @@
+
+import grpc
+import nixnet_pb2 as nixnet_types
+import nixnet_pb2_grpc as grpc_nixnet
+import getch
+
+SERVER_ADDRESS = "localhost"
+SERVER_PORT = "31763"
+
```

Comment:
We should also add a logic to take Server_adress and server_port as command line arguments from the clients. See other examples.

Reply:
Added this logic

Reply:
`pip install ni-python-styleguide` and `pip install black`. 

To execute them:

> ni-python-styleguide --exclude .\source\codegen\metadata\ lint .\source\codegen\ .\examples\

This will show you any errors flagged by ni styleguide

> black source/codegen examples

This command will report any issues flagged by black and fix it as well.

Here's the wiki that has some relevant information
[grpc-device (Teams) -> wiki](https://teams.microsoft.com/l/entity/com.microsoft.teamspace.tab.wiki/tab::7426a53d-3098-4c17-9bb0-5ba77dc51c7f?context=%7B%22subEntityId%22%3A%22%7B%5C%22pageId%5C%22%3A7%2C%5C%22origin%5C%22%3A2%7D%22%2C%22channelId%22%3A%2219%3A658a48f1bcd240dc9c56f5eb35761898%40thread.tacv2%22%7D&tenantId=87ba1f9a-44cd-43a6-b008-6fdb45a5204e)